### PR TITLE
Re-enable the ability to use a custom refresh rate on Windows 11

### PR DESCRIPTION
**Pull request by 0xENDER**
Note: I couldn't test this locally, so you might want to test this first before accepting the pull request.

### CODE REVIEW

**Comment by 0xENDER** [comment]
#78

**Comment by Seo-Rii** [comment]
Is there still a delay related to drag in Windows 11?

File: src/os.ts [hunk 1]
```diff
@@ -1,11 +1,7 @@
 import * as os from 'os'
 
 export const isWindows10OrGreater =
-	process.platform === 'win32' && os.release().split('.')[0] === '10'
-
-export const isWindows11OrGreater =
```
Comment (dancingpizza) [inline src/os.ts:6]: Maybe this method should stay? just in case

**Comment by 0xENDER** [comment]
> Is there still a delay related to drag in Windows 11?

Yes

**Pull request closed without merging**